vumi_http_retry/service.py

```diff
@@ -0,0 +1,23 @@
+from twisted.web import server
+from twisted.python import usage
+from twisted.application import service, strports
+
+from vumi_http_retry.app import VumiHttpRetryServer
+
+
+class Options(usage.Options):
+    optParameters = [[
+        'port', 'p', '8080',
+        'Port to listen on'
+    ]]
+
+
+def makeService(options):
+    server = VumiHttpRetryServer()
+
+    site = server.Site(server.app.resource())
+    srv_service = service.MultiService()
```

Comment:
I just copy-pasted what diamondash was doing here, happy to use something else that makes more sense.